File: exercicios_py/secao7/exercicio5.py
#Faça um programa que leia um nome de usuário e a sua senha e não aceite a senha
#igual ao nome do usuário, mostrando um erro e voltando a pedir informações.

uname = str(input("Digite seu nome de usuário: "))
passw = str(input("Digite sua senha: "))

while uname == passw:
    print ("Erro!")
    uname = str(input("Digite seu nome de usuário: "))
    passw = str(input("Digite sua senha: "))
    

File: exercicios_java/exercicios_java/src/secao7/exercicio5.java
//Faça um programa que leia um nome de usuário e a sua senha e não aceite a senha
//igual ao nome do usuário, mostrando um erro e voltando a pedir informações.

package secao7;
import java.util.Scanner;

public class exercicio5{
    public static void main(String[] args){
        String user, passw;
        Scanner t = new Scanner(System.in);

        System.out.print("Digite sua senha: ");
        user = t.next();
        System.out.print("Digite seu nome de usuário: ");
        passw = t.next();

        while(user.equals(passw)){ //o método equals é usado no java para comparações entre strings
            System.out.print("Digite sua senha: ");
            user = t.next();
            System.out.print("Digite seu nome de usuário: ");
            passw = t.next();
        }  
        t.close();     
    }
}
